Bottom: 9390b4918491f99071f2e612482d91b54658881a
Top:    b891683367002221263337f77519b98242b329d3
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-07 10:56:05 +0800

Refresh of autofs-5.0.7-teach-dumpmaps-to-output-simple-key-value-pairs.patch

---

diff --git a/man/automount.8 b/man/automount.8
index 9b2b9f0..48f5c5a 100644
--- a/man/automount.8
+++ b/man/automount.8
@@ -70,9 +70,9 @@ If the map is an LDAP map and there is more than one map of same name
 in different base dns only the first map encountered by autofs will
 be listed.
 
-If the map type is an old style multi map and any one of the map
-names matches the entries that would be used by autofs for all
-the maps in the multi-map will be listed.
+If the map type is an old style multi-map and any one of the map
+names in the multi-map entry matches the given map name the entries
+that would be used by autofs for the whole multi-map will be listed.
 .TP
 .I "\-O, \-\-global-options"
 Allows the specification of global mount options used for all master
